css/components/Avatar.css

```diff
@@ -0,0 +1,28 @@
+.rounded-avatar,.retina-avatar{
```

Comment:
Here I believe `avatar` class name can be used and one retina-avatar class style can be placed as default style.

Reply:
resolved